cypress/features/regression/test/actionPopoverEvents.feature

```diff
@@ -3,9 +3,9 @@ Feature: Action Popover component
 
   Background: Open Action Popover component page
     Given I open "Test Action Popover" component page
-      And I click the menu button element with iFrame
+      And I click the menu button element in noIFrame
```

Comment:
it seems to be open in iFrame ? - Given step

Reply:
not all TC have that step